src/main/scala/mesosphere/marathon/api/MarathonExceptionMapper.scala

```diff
@@ -70,6 +70,8 @@ class MarathonExceptionMapper extends ExceptionMapper[Exception] {
       )
     case e: JsonParseException =>
       Json.obj("message" -> e.getOriginalMessage)
+    case e: JsonMappingException =>
+      Json.obj("message" -> "Please specify data in JSON format")
```

Comment:
You introduced this to deal with parsing messages which referenced to "Source@234234234" or something like that, correct?

Even though the message was bad, it contained some more information since it actually said where the parse error was in the input, e.g. like this:

```
{
    "message": "Unexpected close marker '}': expected ']' (for ARRAY starting at [Source: [B@534b9d7e; line: 1, column: 0])"
}
```

It would be nice if you preserved that info.